scraper/README.md

```diff
@@ -19,20 +19,54 @@ npm install && mkdir -p data
 cp .env.example .env
 ```
 
-`PRACTICALPLANTSPATH` is the path on your local filesystem to the Practical Plants wiki dump, that could be fetched from our [repository](https://github.com/ElektraInitiative/practicalplants). This is required to scrape the data.
+`PRACTICALPLANTSPATH` mentioned in .env is the path on your local filesystem to the Practical Plants wiki dump, that could be fetched from our [repository](https://github.com/ElektraInitiative/practicalplants). This is required to scrape the data.
 
 3. Start the scraper
 
+The scraper will scrape the data from the Practical Plants wiki dump and store it in `csv` format in the `data` directory. This can be done with the following command:
+
 ```shell
 npm run start
 ```
 
-4. Insert data into database
+As an additional step the script will also lookup the german common name through Wikidata API and store it under the column `Column Name DE` in the `detail.csv` file.
 
-After the data is scraped and stored in `csv` format, we should now insert it into the database. This can be done with the following command:
+There will be several CSV files generated in the `data` directory:
 
-_Note:_ Please make sure that the database is up-to-date with latest migrations before running this command.
+-   `detail.csv`: This file contains the data scraped from the Practical Plants wiki dump.
```

Comment:
Please specify these files (which are CC licensed) in the file [.reuse/dep5](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5)